crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -468,7 +460,9 @@ def services
           false,
           services: {
             data: e.message,
-            help: "Crowbar has failed. Check /var/log/crowbar/production.log for details."
+            help: "Check /var/log/crowbar/production.log at admin server for details." \
+              "If the action failed at a specific node, " \
+              "check /var/log/crowbar/node-upgrade.log at the node."
```

Comment:
Nit: Isn't it `s/at/on/`?

And  `check /var/log/crowbar/node-upgrade.log *on that* node.` ?